cmd/db_audit_logs_exporter_test.go

```diff
@@ -68,17 +78,17 @@ var _ = Describe("Db Audit", func() {
 			Expect(err).ToNot(HaveOccurred())
 			server.AppendHandlers(
 				ghttp.CombineHandlers(
-					ghttp.VerifyRequest(http.MethodPost, "/api/public/v1/accounts/340af43a-8a7c-4659-9258-4876fd6a207b/projects/78d4459c-0f45-47a5-899a-45ddf43eba6e/clusters/7bb68af6-0875-42e0-8665-dcf634ed9fd1/db-audit-log-exporter-configs"),
+					ghttp.VerifyRequest(http.MethodPost, "/api/public/v1/accounts/340af43a-8a7c-4659-9258-4876fd6a207b/projects/78d4459c-0f45-47a5-899a-45ddf43eba6e/clusters/5f80730f-ba3f-4f7e-8c01-f8fa4c90dad8/db-audit-log-exporter-configs"),
```

Comment:
Are the mentioned accound id, project id, cluster id expected to be hardcoded @Rohita2002? Asking since I am not well versed with the CLI code changes.

Reply:
these are test files, the ids are taken from a sample test file which simulates a call to apiserver and just use the response from a json file